docs/api/app.md

```diff
@@ -1222,7 +1222,7 @@ Show the app's about panel options. These options can be overridden with `app.se
   * `credits` String (optional) _macOS_ _Windows_ - Credit information.
   * `authors` String[] (optional) _Linux_ - List of app authors.
   * `website` String (optional) _Linux_ - The app's website.
-  * `iconPath` String (optional) _Linux_ _Windows_ - Path to the app's icon. On Linux, will be shown as 64x64 pixels while retaining aspect ratio.
+  * `iconPath` String (optional) _Linux_ _Windows_ - Path to the app's icon with `.jpg` or `.png` extensions (`.ico` doesn't supported). On Linux, will be shown as 64x64 pixels while retaining aspect ratio.
```

Comment:
I think it's sufficient to say: `Path to the app's icon with `.jpg` or `.png` extensions`. It's clearly implied that other formats are therefore unsupported.

Reply:
Perhaps `in a JPEG or PNG file format` as opposed to mentioning file extensions?